data/docs/messaging-queues/confluent-kafka.mdx

```diff
@@ -0,0 +1,681 @@
+
+**Step 3:** Select metric for Confluent Cloud.
+
+All metrics starting with `confluent_kafka_server_` have been collected from Confluent Cloud.
+
+For example `confluent_kafka_server_received_records` is one of the metrics that was collected.
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/messaging-queues/confluent-kafka/sample-metric.webp"
+        alt="Sample Metric from Confluent Cloud"
+    />
+<figcaption>
+<i>
+Sample Metric from Confluent Cloud
+</i>
+</figcaption>
+</figure>
+
+## Troubleshooting
+
+If you run into any problems while setting up monitoring for your Confluent Cloud metrics with SigNoz, consider these troubleshooting steps:
+
+* **Verify API Key Setup**: Double-check that you have followed the steps to create API Key as shown above.
+* **Verify Configuration**: Double-check your `config.yaml` file to ensure all settings, including the ingestion key and endpoint, are correct.
+* **Review Logs**: Look at the logs of the OpenTelemetry Collector to identify any error messages or warnings that might provide insights into what’s going wrong.
+* **Consult Documentation**: Review the Confluent Cloud, SigNoz and OpenTelemetry documentation for any additional troubleshooting of the common issues.
+
+</TabItem>
+<TabItem value="confluent-platform" label="Confluent Platform">
+
+## Steps to Follow
+
+This guide follows these primary steps:
+
+1. [Confluent Platform Setup](#step-1-confluent-platform-setup)
+2. [OpenTelemetry Java Agent Installation](#step-2-opentelemetry-java-agent-installation)
+3. [Java Producer-Consumer App Setup](#step-3-java-producer-consumer-app-setup)
+4. [OpenTelemetry Collector Setup](#step-4-opentelemetry-collector-setup)
+5. [Visualize data in SigNoz](#step-5-visualize-data-in-signoz)
+
+### Step 1: Confluent Platform Setup
+
+#### 1.1 Confluent Self-Managed Setup
+
+Go to [Confluent Get Started](https://www.confluent.io/get-started/) page, and select **SELF-MANAGED**. Fill in the appropriate details, and click on **Download Free**.
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/messaging-queues/confluent-kafka/confluent-self-managed.webp"
+        alt="Confluent Self-Managed"
+    />
+<figcaption>
+<i>
+Confluent Self-Managed
+</i>
+</figcaption>
+</figure>
+
+In the **Choose An Installation Type** page that opens, download the appropraite installation as per your requirement. 
+In this case, we will be downloading tar installation for local machine.
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/messaging-queues/confluent-kafka/tar-installation-download.webp"
+        alt="TAR installation download"
+    />
+<figcaption>
+<i>
+TAR installation download
+</i>
+</figcaption>
+</figure>
+
+You can move the downloaded tar file to an appropriate directory if required. Once in the appropriate directory, we will untar the file and cd into the confluent folder.
+
+```bash
+$ tar -xvzf confluent-7.7.1.tar
+$ cd confluent-7.7.1
+```
```

Comment:
Please add a note saying that replace `7.7.1` with the version that you downloaded.